pom.xml

```diff
@@ -372,7 +372,7 @@
           <localRepositoryPath>${basedir}/target/local-repo</localRepositoryPath>
           <settingsFile>src/it/settings.xml</settingsFile>
           <pomIncludes>
-            <pomInclude>*/pom.xml</pomInclude>
+            <pomInclude>*</pomInclude>
```

Comment:
any subdirectory (non recursive) of `src/it`  is now a valid IT - we do not need a `pom.xml` in the root which allows us to have multiple projects for a single IT that do not share a common aggregator